service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C32bDischargeOfCareOrderDocumentParameterGenerator.java

```diff
@@ -55,7 +59,7 @@ public DocmosisParameters generate(CaseData caseData) {
 
     @Override
     public DocmosisTemplates template() {
-        return DocmosisTemplates.ORDER;
+        return DocmosisTemplates.ORDER_V2;
```

Comment:
It would be worth making the changes to the original Order template and making sure all tests pass. During development, the V2 would be useful, however, if we aim to have many orders operate within the same template, they must all be tested as such @SooryaSu